pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -14,71 +14,88 @@ import (
 	"github.com/code-ready/crc/pkg/crc/oc"
 )
 
-func checkBundleCached() (bool, error) {
+var genericPreflightChecks = [...]PreflightCheck{
+	{
+		checkLog: "Checking if oc binary is cached",
+		check:    checkOcBinaryCached,
+		fixLog:   "Caching oc binary",
```

Comment:
using `Log` in this naming is very confusing. it is actually NOT the collected log, but a string used for logging purposes as output. Isn't that a description instead?

Reply:
Sure, I'll change it.